public/index.html

```diff
@@ -20,7 +25,7 @@ <h1 id="title">Testnet faucet</h1>
             <button type="submit" id="requestTokens" class="request-tokens-button">Request 0.05 G&ouml;ETH</button>
         </div>
         <div class="text-xs-center">
-          <div id="recaptcha" class="g-recaptcha" data-sitekey=""></div>
```

Comment:
Correct. Since we're getting it via the config.json file, we need to make sure that recaptcha is only initialized afterwards, and not when the DOM is ready.